db/migrate/20131004141959_generalize_wiki_menu_items.rb

```diff
@@ -32,8 +32,8 @@ def up
     rename_table :wiki_menu_items, :menu_items
     add_column :menu_items, :type, :string
     rename_column :menu_items, :wiki_id, :navigatable_id
-    rename_index :menu_items, 'index_wiki_menu_items_on_parent_id', 'index_menu_items_on_parent_id'
-    rename_index :menu_items, 'index_wiki_menu_items_on_wiki_id_and_title', 'index_menu_items_on_navigatable_id_and_title'
+    # rename_index :menu_items, 'index_wiki_menu_items_on_parent_id', 'index_menu_items_on_parent_id'
+    # rename_index :menu_items, 'index_wiki_menu_items_on_wiki_id_and_title', 'index_menu_items_on_navigatable_id_and_title'
```

Comment:
Those should not stay uncommented and can be removed:

> In Rails 4.0 when a column or a table is renamed the related indexes are also renamed. If you have migrations which rename the indexes, they are no longer needed.

[Rails upgrade guide](http://guides.rubyonrails.org/upgrading_ruby_on_rails.html)